models/user.py

```diff
@@ -1,38 +1,72 @@
-from models.BaseModel import BaseModel
+from utility.helpers import query
+from flask import g
 
 
-class User(BaseModel):
-
+####
+class User():
+    
     def __init__(self):
-        self.email_userid = {}
-        super().__init__()
+        pass
+
+    def add(self, client_name, email, password):
+
+        return query('user/register',
+              vals=(client_name, email, password))
 
-    def add(self, entry):
-        userid = super().add(entry)
-        email = entry['email']
-        self.email_userid[email] = userid
 
     def validate(self, email, password):
 
-        if self.old_user(email):
-            userid = self.email_userid[email]
-            entry = self.find_val(userid)
-            return entry['password'] == password
+        cur = query('user/find_by_email', vals=(email,))
+
+        row = cur.fetchone()
+
+        if row is None or row == []:
+            return False
         else:
-            False
+            return password == row['passkey']
+
 
     def old_user(self, email):
```

Comment:
Why do you need this function at all? Isn't it equivalent of just calling `find_by_email` and checking the result? Let's delete this.